CMakeLists.txt

```diff
@@ -109,10 +109,40 @@ install(TARGETS minizip EXPORT minizip-exports
   LIBRARY DESTINATION "lib"
   ARCHIVE DESTINATION "lib")
 
+set(INSTALL_CMAKE_DIR "cmake")
 install(EXPORT minizip-exports
-        DESTINATION "cmake"
+        DESTINATION "${INSTALL_CMAKE_DIR}"
         NAMESPACE "MINIZIP::")
 
+# Create and install CMake package config version file to allow find_package()
+
+include(CMakePackageConfigHelpers)
+write_basic_package_version_file(
+  "${CMAKE_CURRENT_BINARY_DIR}/minizipConfigVersion.cmake"
+  VERSION 1.2
+  COMPATIBILITY SameMajorVersion)
+
+# Create config for find_package()
+
+file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/minizipConfig.cmake.in"
+"
+@PACKAGE_INIT@
+include(CMakeFindDependencyMacro)
+find_dependency(ZLIB REQUIRED)
```

Comment:
One thing, I just found out is, that `find_dependency` should not be called with `REQUIRED`, because this is supposed to get forwarded from the consuming code.

I will change this later.

Reply:
Removed `REQUIRED`.